Jenkinsfile

```diff
@@ -279,11 +279,15 @@ pipeline {
     PUBLIC_DOCKER_REGISTRY = 'docker.packages.nuxeo.com'
     MAVEN_OPTS = "$MAVEN_OPTS -Xms512m -Xmx3072m"
     MAVEN_ARGS = getMavenArgs()
-    VERSION = getVersion()
+    VERSION = "11.1-SNAPSHOT"
     DOCKER_TAG = getDockerTagFrom("${VERSION}")
     CHANGE_BRANCH = "${env.CHANGE_BRANCH != null ? env.CHANGE_BRANCH : BRANCH_NAME}"
     CHANGE_TARGET = "${env.CHANGE_TARGET != null ? env.CHANGE_TARGET : BRANCH_NAME}"
     CONNECT_PREPROD_URL = 'https://nos-preprod-connect.nuxeocloud.com/nuxeo'
+    ORG = "nuxeo"
+    APP_NAME = "nuxeo"
+    NAMESPACE = "$APP_NAME-${BRANCH_NAME.toLowerCase()}"
```

Comment:
I'd go with `PREVIEW_NAMESPACE` so it is explicit.
We're also using some namespaces for unit tests, for which the related env vars are prefixed by `TEST_`, see for instance https://github.com/nuxeo/nuxeo/blob/master/Jenkinsfile#L267.

Reply:
About ORG and APP_NAME cf ttps://github.com/nuxeo/nuxeo/pull/3856#discussion_r400105125
About renaming NAMESPACE to TEST_NAMESPACE
There is a testNameSpace local variable in buildUnitTestStage.
it's distinguishable but yet buggling.

Reply:
https://github.com//pull/3856#discussion_r400105125 leads to a 404.
APP_NAME is used yes, just don't see ORG.

Yes, please rename NAMESPACE to PREVIEW_NAMESPACE so we know it' the one used for preview.
There are plenty of namespaces involved: the one the pod is running in, the default one, the test ones, ...

Reply:
OK, about ORG, the explanation is that `jx preview` is requiring it [here](https://github.com/jenkins-x/jx/blob/8fdc3a1182bc3ed6ae09728721b53ea4fa1d6234/pkg/cmd/preview/preview.go#L1007), though it isn't documented...
And there's a closed issue though it wasn't really resolved: https://github.com/jenkins-x/jx/issues/1992.